Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -103,274 +49,7 @@ static void drawString(UIFont* font,
                        UITextAlignment alignment,
                        CGSize* sizeOut,
                        CGPoint* lastCharPos = NULL) {
-    if (font == nil) {
```

Comment:
#1132 deals with this, but I needed to get rid of this locally to unbreak some internal test apps.